src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -60,7 +63,13 @@
     public static final CharMatcher allowedProperties = CharMatcher.JAVA_LETTER_OR_DIGIT.or(CharMatcher.anyOf(ALLOWED_CHARS));
     private static Configuration PARENT = null;
 
-    File file;
+    @Deprecated
+    private File file;
+
+    @Deprecated
+    private CharSource staticSource;
+    @Deprecated
+    private CharSink staticSink;
```

Comment:
Why are all these deprecated? That does barely make sense on private fields and even less on *newly added* private fields.